db-schema/mongo/AuthorDetail.js

```diff
@@ -2,20 +2,22 @@ import { mongoose } from '../../db/mongoose.js'
 
 
 const authorDetailSchema = new mongoose.Schema({
-    authorId: {
```

Comment:
@MukundSprinto There seems to be diff from empty, is `empty` actually empty branch?